addons/stock_account/views/stock_valuation_layer_views.xml

```diff
@@ -146,6 +151,7 @@
                 <filter string="Has Remaining Qty" name="has_remaining_qty" domain="[('remaining_qty', '>', 0)]"/>
                 <group expand='0' string='Group by...'>
                     <filter string='Product' name="group_by_product_id" context="{'group_by': 'product_id'}"/>
+                    <filter string='Lot/Serial number' name="group_by_lot_id" context="{'group_by': 'lot_id'}"/>
```

Comment:
Hi @Whenrow (: Amazing job on this massive new feature! 

I'm a technical writer from the SF office. I'm not sure if it's my place to point this out, but I was wondering if we could capitalize "Number" here, to maintain consistency in capitalization with existing filters in this drop-down menu.

![image](https://github.com/user-attachments/assets/c79b6f75-1f3e-464d-bc0f-cfbeb3476455)
```suggestion
                    <filter string='Lot/Serial Number' name="group_by_lot_id" context="{'group_by': 'lot_id'}"/>
```